project/src/main/java/com/google/sps/data/errors/FetcherException.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.errors;
+/**
+ * FetcherException represents an error returned by the PlacesFetcher.
+ */
+public class FetcherException extends Exception {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
Do you have to declare this? 
I don't think you expect to serialize this Exception anytime soon..

Reply:
Yes it probably won't be serialized haha.. I added it because I got a warning that this should be declared. I added an annotation so I won't get that warning and removed the serialVersionUID declaration.